### DISCUSSION TITLE
Change background color on macOS

### DISCUSSION THREAD

**Discussion by yashh** [ghostty-org/ghostty#3198] [Q&A] [answered]
```
background = 000000
foreground = f4f4ff
cursor-color = ff0078
```

The background value is ignored in macOS Ghostty version 1.0. The default background color is R: 40, G: 44, B: 52 which is specified in the zig code. Does this mean it does not work in macOS or is there some setting to unlock this?

Also cmd+F does not give any search interface. Is this not implemented and in the roadmap?
![Screenshot 2024-12-26 at 4 01 02 PM](https://github.com/user-attachments/assets/9cc4e0ef-71c5-4911-b701-65634ca0834a)

**Comment by mitchellh** [comment]
Search: #189 

The background color: did you reload your config?

**Reply by yashh** [reply]
Yes. I tried `Shift + Cmd + ,` and also closed Ghostty and restarted it. No effect.

**Reply by mitchellh** [reply, marked as answer]
It works for me. Are you sure your config file is in the correct location? https://ghostty.org/docs/config#file-location

**Reply by yashh** [reply]
Ok I had to delete the config file at `~/Library/Application\ Support/com.mitchellh.ghostty/config`  which then made the `~/.config/ghostty/config` the only config. I think the issue was since background existed in the ~Library one we need to make background empty and then set it to a new value. 

Anyway thank you @mitchellh. You have done a great job on Ghostty.
